### PR TITLE
Gallery submission

### PR DESCRIPTION
**Pull request by Jessonsotoventura**
Finished first draft of the page, missing copy to clipboard and link functionality

### CODE REVIEW

**Review by memo330179 — CHANGES_REQUESTED** [review]
Image needs to be moved to proper place

File: app/controllers/gallery_submission.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
-from app.allImports import *
+from app.allImports import * 
 from app.logic.validation import *
 from werkzeug.security import check_password_hash
+from app.logic.queries import GalleryQueries
```
Comment (memo330179) [inline app/controllers/gallery_submission.py]: After `GalleryQueries.py` is moved don't forget to change import to reflect new location

File: app/logic/queries/GalleryQueries.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from app.models.Galleries import *
```
Comment (memo330179) [inline app/logic/queries/GalleryQueries.py]: This file needs to be moved to `app/models/queries`

**Review by memo330179 — CHANGES_REQUESTED** [review]
We don't store images in data folder. This should be in app/static/data

File: scripts/ConfigureApp.py [hunk 3]
```diff
@@ -167,49 +112,27 @@ def add_dummy_data_prompt(self):
...
+        self.create_mysql_database()
 
 
 
 if __name__ == '__main__':
     create = ConfigureApp()
     create.main()
 
+
 #TODO:
```
Comment (memo330179) [inline scripts/ConfigureApp.py:136]: Is this TODO: still relevant?

**Review by memo330179 — APPROVED** [review]

File: scripts/ConfigureApp.py (hunk 3 above)
Comment (Jessonsotoventura) [reply scripts/ConfigureApp.py:136]: No I'll get rid of it soon

**Pull request merged**